src/libANGLE/renderer/d3d/d3d11/ExternalImageSiblingImpl11.cpp

```diff
@@ -160,52 +202,55 @@ angle::Result ExternalImageSiblingImpl11::createRenderTarget(const gl::Context *
 
         ANGLE_TRY(mRenderer->allocateResource(context11, rtvDesc, mTexture.get(), &rtv));
         rtv.setInternalName("getAttachmentRenderTarget.RTV");
+        
+        mRenderTarget = std::make_unique<TextureRenderTarget11>(
+            std::move(rtv), mTexture, std::move(srv), std::move(blitSrv), mFormat.info->internalFormat,
+            formatInfo, mWidth, mHeight, 1, mSamples);
     }
-
-    d3d11::SharedSRV srv;
-    if (mIsTexturable)
```

Comment:
nit: keeping SRV creation for last is likely to yield a less conflict-prone diff

Reply:
When we talked about this I couldn't remember why I swapped the order, but since then I figured it out: The `SRV` is needed for both RenderTarget constructor call and I moved these inside of their corresponding if, therefore the `SRV` creation must precede those.